packages/coinstac-common/src/models/consortium.js

```diff
@@ -53,6 +53,17 @@ class Consortium extends PouchDocument {
 Consortium.schema = Object.assign({}, PouchDocument.schema, {
   _id: joi.string().regex(/^[a-zA-Z0-9]+$/),
   activeComputationId: joi.string().min(1),
+
+  /**
+   * @todo A computation's inputs shouldn't be set on the consortium because
+   * this limits runs to one-at-a-time. Consider moving elsewhere, perhaps some
+   * sort of `ComputationRun` model.
+   */
+  activeComputationInputs: joi.array().items(
```

Comment:
Owners select inputs in the UI, then save them here. These are serialized into the local and remote doc for each run, so `activeComputationInputs` can be changed for the next run.